.gitignore

```diff
@@ -9,3 +9,6 @@ __pycache__/
 venv/
 *.egg-info/
 dist/
+
+# for local types testing
+/test_script.py
```

Comment:
please remove this one,
if you use the local script for dev-testing, just don't add it into git
in gitignore we add the files that created automaticaly when python running and doesn't needed for code working in new machine